### DISCUSSION TITLE
Following counter is wrong

### DISCUSSION THREAD

**Discussion by CyberPingU** [pixelfed/pixelfed#4821] [Q&A]
Hello all,
I brought up a couple of instances: a user on one instance is following another user of the same instance and a second user from the other instance. Anyhow in his below his avatar there's written only "1 following"; if I click on the profile, I can see that the user is actually having 2 following in its analythic list.

How to correct this?

**Comment by splacat** [comment]
Yes, I'm running last Pixelfed version and I see this wrong counter behaviour too.
